docs/conf.py

```diff
@@ -298,3 +299,15 @@ def setup(app):
 
 # sphinx-tabs configuration
 sphinx_tabs_disable_tab_closing = True
+
+# sphinxext-opengraph configuration
+
+ogp_image = '_images/logo.png'
+ogp_use_first_image = True
+ogp_enable_meta_description = True
+ogp_description_length = 300
+
+## RTD sets html_baseurl, ensures we use the correct env for canonical URLs
+## Useful to generate correct meta tags for Open Graph
+## Refs: https://github.com/readthedocs/readthedocs.org/issues/10226, https://github.com/urllib3/urllib3/pull/3064
+html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "/")
```

Comment:
Taken [from the docs of RTD](https://github.com/readthedocs/readthedocs.org/blob/ab2f8a013593c8a1eed68e96dc90a3122ee6184c/docs/conf.py#L71-L76). It allows the PR to generate correct meta tags (mostly for images). This page shows how the links would look like: 

https://www.opengraph.xyz/url/https%3A%2F%2Fpostgrest--721.org.readthedocs.build%2Fen%2F721%2Freferences%2Fapi%2Fresource_embedding.html